packages/synapse-react-client/mocks/faker/generateDiscussion.ts

```diff
@@ -0,0 +1,40 @@
+import { DiscussionThreadBundle, Forum } from '@sage-bionetworks/synapse-types'
+import { faker } from '@faker-js/faker'
+import { mockUserData } from '../user/mock_user_profile'
+import { pickRandomMockUser } from './fakerUtils'
+import { SetRequired } from 'type-fest'
+
+export function generateForum(
+  overrides: SetRequired<Partial<Forum>, 'projectId'>,
```

Comment:
When generating a forum, require a project ID, but we'll generate everything else.